### DISCUSSION TITLE
Option for logging exercise?

### DISCUSSION THREAD

**Discussion by DjDenmark** [NightscoutFoundation/xDrip#2033] [General]
Dear developers of xDrip+,
I'm a new user of xDrip+ and do already find the software amazing! I have a feature request regarding adding exercise as a treatment? The LibreLink app has 3 exercise intensities and then you can select the time exercised, which I find super informative when looking at my graphs and estimating insulin dosing in relation to exercise. 
Again thanks for a great product!

**Comment by Navid200** [comment]
This may be related:  https://github.com/NightscoutFoundation/xDrip/issues/1476

**Comment by jonprogers** [comment]
I know this doesn't fully fully cover your request but in the past I have used 2 methods to record exercise which may help you, one is to record in nightscout with the exercise type in the care portal and the other is to just use the freeform notes in xDrip.

**Reply by DjDenmark** [reply]
Thanks for the tip @jonprogers. I still use the LibreLink app where I record it in as well as the freeform notes, but I hope that I could in the future only write it in xDrip. I have not used nightscout yet, but am interested in it now that you mention another way to keep track of it :)

**Comment by Navid200** [comment]
There is a lot about modeling and predictions, the complexities and limitations.  If you go through the open and closed issues, you can find many.  The following are just another example.
https://github.com/NightscoutFoundation/xDrip/issues/91
